### PR TITLE
Add Continuous Integration

### LINKED ISSUE #8: Setup Continuous Integration
Complete this before starting #6, #7, #8, or #5 .

### PR DESCRIPTION
**Pull request by damonkelley**
Resolves #8 

~~Opening this so I can test the travis build. WIP~~

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

**Comment by ldko** [comment]
LGTM :+1:

File: coda/coda_project/settings/test.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from .base import *
+
+SITE_ID = 1
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'coda_local',
+        'USER': os.getenv('DB_MYSQL_USER', default="root"),
```
Comment (somexpert) [inline coda/coda_project/settings/test.py]: let's get rid of those double quotes

**Comment by somexpert** [comment]
:thumbsup:

**Pull request merged**